Runtimes/Core/core/CMakeLists.txt

```diff
@@ -318,6 +318,14 @@ if(NOT POLICY CMP0157)
     $<TARGET_OBJECTS:swiftThreading>)
 endif()
 
+if(WIN32 AND BUILD_SHARED_LIBS)
+  configure_file(swiftCore.1.manifest.in
```

Comment:
This looks like it has a similar goal as `Plist.cmake`. Maybe we can rename that to `InjectMetadata` or something and keep the logic for each platform over there?